---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---
**Check the bug**
Before filling this bug report, please search if a similar issue already exists.
In this case, just add a comment on this existing issue.

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Start Glances with the following options '...'
2. Press the key '....'
3. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environement (please complete the following information)**
 - Operating System (lsb_release -a or OS name/version): `To be completed with result of: lsb_release -a`
 - Glances & psutil versions: `To be completed with result of: glances -V`
 - How do you install Glances (Pypi package, script, package manager, source): `To be completed`
 - Glances test: ` To be completed with result of: glances --issue`

**Additional context**
Add any other context about the problem here.

You can also [pastebin](https://pastebin.com/):

* the Glances configuration file (https://glances.readthedocs.io/en/latest/config.html#location)
* the Glances log file (https://glances.readthedocs.io/en/latest/config.html#logging)
